### ISSUE TITLE
Incompatible with Audio Equalizer web extension

### ISSUE THREAD

**Issue by Gitoffthelawn** [codebicycle/videospeed#111]
Installing the `Audio Equalizer` web extension (https://addons.mozilla.org/firefox/addon/audio-equalizer-wext/) causes this extension to no longer work properly.  Note that Audio Equalizer is a Mozilla recommended extension.

Can they be made compatible?

**Comment by Benzolio** [comment]
I've been trying all combinations of add-ons that offer audio panning and volume per tab with all the add-ons that offer video speed control per tab, and I can't find any combination that works, so I suspect there is something fundamental in the browser so only one add-on can control speed and audio of video playback. I think this one Video Speed Control is the best, the keyboard shortcuts and preferred speed options are awesome. If it is really true that two add-ons can't both control the playback in a tab at the same time, it would be even more awesome if VIdeoSpeed had audio panning and volume built in, too!

**Comment by Gitoffthelawn** [comment]
@BenMarolt Thanks for the feedback.  What you wrote does not match my experiences.  I am able to use Video Speed Controller (this project) together with the SoundFixer extension (https://addons.mozilla.org/firefox/addon/soundfixer/).

Unfortunately SoundFixer has its own bugs and is not nearly as good or flexible as Audio Equalizer (hence, my desire to use VSC with Audio Equalizer).

**Comment by barreira** [comment]
> @BenMarolt Thanks for the feedback. What you wrote does not match my experiences. I am able to use Video Speed Controller (this project) together with the SoundFixer extension (https://addons.mozilla.org/firefox/addon/soundfixer/).

@Gitoffthelawn I have the same issue. Whenever I use an extension that controls the sound (e.g. Audio Equalizer or any volume boost extension) the video speed goes to 1.0x (although the speed shown on video speed controller's controller doesn't change). Trying to set the speed with the video speed controller extension does not do anything thereafter. It only starts working again after refreshing the page (and continues working as long you don't use any sound control extension).

I'm on Firefox 88.0.1 on macOS Big Sur. It works on Chrome, though.

@BenMarolt Have you found any way to make speed control and volume control work at the same time?

**Comment by Gitoffthelawn** [comment]
I wonder if the reason SoundFixer and this extension can co-exist (albeit not functionally well) is because perhaps SoundFixer does not actually do anything unless the user tries to change the volume using its UI.  On the other hand, it's likely Audio Equalizer always tries to modify the sound.

I don't have time right now to go through the code for each of them again, but I have a hunch that may explain the difference.

**Comment by barreira** [comment]
@Gitoffthelawn Might be related to this Firefox bug https://bugzilla.mozilla.org/show_bug.cgi?id=1517199

**Comment by Benzolio** [comment]
> > @BenMarolt Thanks for the feedback. What you wrote does not match my experiences. I am able to use Video Speed Controller (this project) together with the SoundFixer extension (https://addons.mozilla.org/firefox/addon/soundfixer/).
> 
> @Gitoffthelawn I have the same issue. Whenever I use an extension that controls the sound (e.g. Audio Equalizer or any volume boost extension) the video speed goes to 1.0x (although the speed shown on video speed controller's controller doesn't change). Trying to set the speed with the video speed controller extension does not do anything thereafter. It only starts working again after refreshing the page (and continues working as long you don't use any sound control extension).
> 
> I'm on Firefox 88.0.1 on macOS Big Sur. It works on Chrome, though.
> 
> @BenMarolt Have you found any way to make speed control and volume control work at the same time?

No, sorry, and currently no panning extensions are working for me in Firefox 88.0.1
